send_to_s3.py

```diff
@@ -27,15 +29,17 @@ def eprint(*args, **kwargs):
 
     for root, subFolders, files in os.walk(sourcedir):
         for file in files:
-            filename = os.path.join(root,file)
+            filename = os.path.join(root, file)
             filepath = "{0}/{1}".format(filedir, os.path.relpath(filename, sourcedir))
             print("{0} => {1}".format(filename, filepath), file=sys.stderr)
             key = boto.s3.key.Key(bucket, filepath)
-            file_to_send = open(filename, 'r')
+            file_to_send = open(filename, 'rb')
             if (filepath.endswith(".html")):
                 content_type = {"Content-Type": "text/html"}
             elif (filepath.endswith(".svg")):
                 content_type = {"Content-Type": "image/svg+xml"}
+            elif (filepath.endswith(".png")):
+                content_type = {"Content-Type": "image/png"}
```

Comment:
This was mostly just a trial step, it doesn't have any effect right now, but is fine.